internal/services/integrationtesting/testconfigs/arrowtosamesubject.yaml

```diff
@@ -25,5 +25,5 @@ relationships: >-
   folder:folder2#viewer@user:tom
 assertions:
   assertTrue:
-    - 'document:firstdoc#view@user:tom#...'
-    - 'document:seconddoc#view@user:tom#...'
+    - "document:firstdoc#view@user:tom#..."
+    - "document:seconddoc#view@user:tom#..."
```

Comment:
Quotation changes; previously this was something that yamllint yelled about but didn't fix. Prettier happily fixes them.